dnsapi/dns_gandi_livedns.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env sh
+
+# Gandi LiveDNS v5 API
+# http://doc.livedns.gandi.net/
+# currently under beta
+#
+# Requires GANDI API KEY set in GANDI_LIVEDNS_KEY set as environment variable
+#
+#Author: Frédéric Crozat <fcrozat@suse.com>
+#Report Bugs here: https://github.com/fcrozat/acme.sh
+#
+########  Public functions #####################
+
+GANDI_LIVEDNS_API="https://dns.beta.gandi.net/api/v5"
+
+#Usage: dns_gandi_livedns_add   _acme-challenge.www.domain.com   "XKrxpRBosdIKFzxW_CT3KLZNf6q0HG9i01zxXp5CPBs"
+dns_gandi_livedns_add() {
```

Comment:
Gandi DNS is redoing their DNS infrastructure. Their old infrastructure (and API) was making DNS-01 validation very hard to implement (minimum of 20mins for DNS zone push, and sometime, it was even longer). Their new infrastructure (called LiveDNS) is only available as beta, uses a REST api and supports "instant" push of DNS changes. That is why I named it gandi_livedns.. I think they will phase out their "old" API in the near future but I don't know when.

But I can rename it to gandi instead, of course.